src/AnimationManager.ts

```diff
@@ -140,39 +140,42 @@ export class AnimationManager {
 			this._animateParam = { ...param };
 			const info: AnimationParam = this._animateParam;
 			const self = this;
+			const destPos = info.destPos;
+
 			let prevPos = info.depaPos;
 			let prevEasingPer = 0;
 			const directions = map(prevPos, (value, key) => {
-				return value <= info.destPos[key] ? 1 : -1;
+				return value <= destPos[key] ? 1 : -1;
 			});
 			let prevTime = new Date().getTime();
 			info.startTime = prevTime;
 
 			(function loop() {
 				self._raf = null;
 				const currentTime = new Date().getTime();
-				const easingPer = self.easing((currentTime - info.startTime) / param.duration);
-				let toPos: Axis = map(prevPos, (pos, key) => pos + info.delta[key] * (easingPer - prevEasingPer));
+				const ratio = (currentTime - info.startTime) / param.duration;
+				const easingPer = self.easing(ratio);
+				const toPos: Axis = self.axm.map(prevPos, (pos, options, key) => {
+					const nextPos = ratio >= 1
+						? destPos[key]
+						: pos + info.delta[key] * (easingPer - prevEasingPer);
 
-				toPos = self.axm.map(toPos, (pos, options, key) => {
 					// fix absolute position to relative position
```

Comment:
@daybrush It would be better to change comment like follows.

`Subtract distance from distance already moved.`